lib/Raven/Client.php

```diff
@@ -884,6 +884,8 @@ public function capture($data, $stack = null, $vars = null)
         if (empty($data['site'])) {
             unset($data['site']);
         }
+        $data['contexts']['runtime'] =  array('version' => phpversion(), 'name' => 'php');
```

Comment:
The blank line added below this line should probably go above. Also, maybe it would be better to have the `php` string in uppercase. Last thing, the most important: please use the `PHP_VERSION` constant as it's faster because we don't have the overhead of a function call

Reply:
'php' should not be uppercased as this is a value sent to the Sentry server and is reflective on the actual platform (it generally should always be lowercased)

Reply:
That string is strictly validated on server side, so we cannot change it, see [the docs about this](https://docs.sentry.io/clientdev/attributes/#required-attributes) and the related [API JSON schema](https://sentry.io/api/store/schema). 
Lowercase it is then.